LondonBritishLibrary/orient/BLorient814.xml

```diff
@@ -209,6 +84,233 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             </recordHist>
                         </adminInfo>
                     </additional>
+                    <msPart xml:id="p1">
+                      <msIdentifier></msIdentifier>
+                      <msContents>
+                        <msItem xml:id="p1_i1">
+                           <locus target="#2 #4"/>
```

Comment:
I think that if the correct order is 4, 2, this could be encoded as
```
 <locus target="#4"/>
 <locus target="#2"/>
```
which would indicate the order already here.